docs/SalesforceMultiObjects-batchsource.md

```diff
@@ -158,3 +158,43 @@ multisink.Activity =
     ]
   }
 ```
```

Comment:
Is the **Cases when BULK API is not used:** section also applicable to multi object source? If yes, can you please add it here?

Reply:
Yes, this section is also applicable to multi object source and it's already added in the documentation.